### PR TITLE
[DRAFT] KN workflow plugin installation

### PR DESCRIPTION
**Pull request by wmedvede**
Closes/Fixes/Resolves #ISSUE-NUMBER

**Description:**

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `Issue-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] Issue-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+== Installing the Knative Workflow Plugin using the artifacts image
+
+To install the Knative Workflow Plugin using the artifacts image you must follow this procedure:
+
+*Start the `kn-workflow-cli-artifacts-rhel8` image*
+
+[source, shell]
+----
+export KN_IMAGE=registry.redhat.io/openshift-serverless-1/kn-workflow-cli-artifacts-rhel8:1.33.0
```
Comment (wmedvede) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc:22]:
```suggestion
export KN_IMAGE=registry.redhat.io/openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+// Metadata:
+:description: Install the operator on Kubernetes clusters
+:keywords: kogito, sonataflow, workflow, serverless, operator, kubernetes, minikube, openshift, containers
+// links
+
+*Prerequisites*
+
+https://75129--ocpdocs-pr.netlify.app/openshift-serverless/latest/install/installing-kn
+
+* You have first installed the  link:https://76490--ocpdocs-pr.netlify.app/openshift-serverless/latest/install/installing-kn[Knative CLI].
```
Comment (domhanak) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc:12]: This should be a link to official community documentation of knative

Comment (wmedvede) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/install-kn-workflow-cli.adoc:12]: yes, we can change that link.
But please note that this PR is just a draft to share what the proposal of installing the kn workflow plugin was. IDK if we are publishing that image in community·

**Comment by domhanak** [comment]
Closing now, thanks @wmedvede for the effort.

**Pull request closed without merging**